packages/esm-bed-management-app/src/bed-administration/bed-administration-form.component.tsx

```diff
@@ -265,13 +263,13 @@ const BedAdministrationForm: React.FC<BedAdministrationFormProps> = ({
             </FormGroup>
             {showErrorNotification && (
               <InlineNotification
+                kind="error"
                 lowContrast
-                title={t('error', 'Error')}
-                style={{ minWidth: '100%', margin: '0', padding: '0' }}
+                onClose={() => setShowErrorNotification(false)}
                 role="alert"
-                kind="error"
+                style={{ minWidth: '100%', margin: '0', padding: '0' }}
```

Comment:
Future work: I think we should (in general) prefer classes rather than inline styles wherever possible.

Reply:
Yeah, this will be easier to do once https://openmrs.atlassian.net/browse/O3-4199 is in.